### DISCUSSION TITLE
How can i solve this problem?

### DISCUSSION THREAD

**Discussion by Batgerel53** [saleor/saleor#8699] [Q&A]
![gql](https://user-images.githubusercontent.com/56902009/144369627-d8d8ab23-85c1-4ac5-8262-7d8a3360d0dc.PNG)

There are many products data in database but there is no products cant be call on graphql

**Comment by Batgerel53** [comment]
@patrys  @karolkielecki  @maarcingebala

**Reply by karolkielecki** [reply]
@Batgerel53 Did you assign any products to that channel? You use filter `channel`.

**Comment by Batgerel53** [comment]
![dnd](https://user-images.githubusercontent.com/56902009/144526124-ba38b619-8f76-44f8-95e9-d9f877b44178.PNG)

**Reply by Batgerel53** [reply]
@karolkielecki

**Reply by Batgerel53** [reply]
![error1PNG](https://user-images.githubusercontent.com/56902009/144528568-f109ecbf-2880-4b47-af80-427e4112f27f.PNG)

**Reply by karolkielecki** [reply]
Which version are you using? @Batgerel53
